Refit.Tests/RequestBuilder.cs

```diff
@@ -221,6 +223,19 @@ public void PostWithObjectQueryParameterHasCorrectQuerystring()
             Assert.Equal("/foo?test-query-alias=one&TestAlias2=two", uri.PathAndQuery);
         }
 
+        [Fact]
+        public void ObjectQueryParameterWithInnerCollectionHasCorrectQuerystring()
+        {
+            var fixture = new RequestBuilderImplementation<IDummyHttpApi>();
+            var factory = fixture.BuildRequestFactoryForMethod(nameof(IDummyHttpApi.ComplexTypeQueryWithInnerCollection));
+
+            var param = new ComplexQueryObject { TestCollection = new[] { 1, 2, 3 } };
```

Comment:
What about `new []{1.1,2.1,3.0}`?
in some locale like `pt ` and `pt-br` `(1.2).ToString()` results in `1,2`.
We need to make sure that CultureInfo.InvariantCulture is used

Reply:
Hmm. I didn't modify any code that actually converts properties to their string representation, but it looks like `CultureInfo.InvariantCulture` is already being used in the relevant section https://github.com/reactiveui/refit/blob/00b21bf7e25bab5f930d8b0abebdcf7d118f9c3c/Refit/RefitSettings.cs#L86 and there's a test around the culture issue as well https://github.com/reactiveui/refit/blob/00b21bf7e25bab5f930d8b0abebdcf7d118f9c3c/Refit.Tests/RequestBuilder.cs#L1564

Are you having a specific problem with this?